common/defs/bitcoin/bitcoin_regtest.json

```diff
@@ -0,0 +1,43 @@
+{
+  "coin_name": "Regtest",
+  "coin_shortcut": "REGTEST",
+  "coin_label": "Regtest",
+  "website": "https://bitcoin.org",
+  "github": "https://github.com/bitcoin/bitcoin",
+  "maintainer": "Pavol Rusnak <stick@satoshilabs.com>",
```

Comment:
please change the maintainer - you are submitting the file and you are responsible for its contents

Reply:
done